qiskit_experiments/library/calibration/rough_drag_cal.py

```diff
@@ -0,0 +1,116 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""Rough drag calibration experiment."""
+
+from typing import Iterable, List, Optional
+
+from qiskit import QuantumCircuit
+from qiskit.circuit import Parameter
+from qiskit.providers.backend import Backend
+
+from qiskit_experiments.framework import ExperimentData, fix_class_docs
+from qiskit_experiments.calibration_management import (
+    BaseCalibrationExperiment,
+    BackendCalibrations,
+)
+from qiskit_experiments.calibration_management.update_library import BaseUpdater
+from qiskit_experiments.library.characterization.drag import RoughDrag
+
+
+@fix_class_docs
+class RoughDragCal(BaseCalibrationExperiment, RoughDrag):
```

Comment:
Is this `RoughDragXSXCal` as per convention of rough amp?

Reply:
No, Drag only ever updates one pulse. To update both X and SX you should run two separate experiments with the corresponding pulses.

Reply:
Probably we can integrate them in future since rough drag doesn't have enough resolution of beta.